hal/src/nRF52840/deviceid_hal.cpp

```diff
@@ -111,6 +112,21 @@ int hal_get_device_hw_model(uint32_t* model, uint32_t* variant, void* reserved)
     return SYSTEM_ERROR_NONE;
 }
 
+int hal_get_device_hw_info(hal_device_hw_info* info, void* reserved) {
```

Comment:
Does `info->size` need to be handled in this function?

Reply:
Not as of right now. Only once this struct is extended.